frontend/public/components/overview/resource-overview-page.tsx

```diff
@@ -35,8 +35,11 @@ const getPluginTabSectionResource = (item) => {
 export const OverviewDetailsResourcesTab: React.SFC<OverviewDetailsResourcesTabProps> = ({
   item,
 }) => {
+  const [pluginComponents, setPluginComponents] = React.useState([]);
   const { buildConfigs, routes, services, pods, obj } = item;
-  const pluginComponents = getPluginTabSectionResource(item);
+  React.useEffect(() => {
+    setPluginComponents(getPluginTabSectionResource(item));
+  }, []);
```

Comment:
```
const pluginComponents = React.useMemo(() => getPluginTabSectionResource(item), [item]);
```

Reply:
I have tried it, the problem is the item changes frequently as it holds a lot of resources in it. Eveytime the item changes it recreates/lazy loads the component again and again. These components should be mounted only once and thats why i added it in useEffect.

Reply:
Can we use the hook `usePluginsOverviewTabSection` here?

Reply:
updated usePluginsOverviewTabSection hook and used it here.